vnext/Microsoft.ReactNative/RedBox.cpp

```diff
@@ -151,8 +164,54 @@ struct RedBox : public std::enable_shared_from_this<RedBox> {
         m_errorMessageText.Text(Microsoft::Common::Unicode::Utf8ToUtf16(message));
         m_errorStackText.Text(Microsoft::Common::Unicode::Utf8ToUtf16(stack));
         return;
+      } else if (json.count("type") && json["type"] == "InternalError") {
+        auto message = json["message"].asString();
+        m_errorMessageText.Text(Microsoft::Common::Unicode::Utf8ToUtf16(message));
+
+        if (IsMetroBundlerError(message, json["type"].asString())) {
+          m_errorStackText.Text(
+              Microsoft::Common::Unicode::Utf8ToUtf16(json["type"].asString() + (" ─ See " METRO_TROUBLESHOOTING_URL)));
+          m_errorStackText.IsTextSelectionEnabled(false);
+          m_errorStackText.Tapped([](IInspectable const & /*sender*/, xaml::Input::TappedRoutedEventArgs const &e) {
+            winrt::Windows::System::Launcher::LaunchUriAsync(Uri(MAKE_WIDE_STR(METRO_TROUBLESHOOTING_URL)));
+          });
+
+          m_errorStackText.PointerEntered([&](IInspectable const &, xaml::Input::PointerRoutedEventArgs const &) {
+            auto hand = winrt::Windows::UI::Core::CoreCursor(winrt::Windows::UI::Core::CoreCursorType::Hand, 1);
+            m_beforeCursor = xaml::Window::Current().CoreWindow().PointerCursor();
+            xaml::Window::Current().CoreWindow().PointerCursor(hand);
```

Comment:
Please dont add additional dependencies on CoreWindow.  They dont work in XAML islands.

Reply:
You could use a Hyperlink instead.  It's an Inline which you could add to TextBlock.Inlines.

---
In reply to: [407142204](https://github.com/microsoft/react-native-windows/pull/4575#discussion_r407142204) [](ancestors = 407142204)

Reply:
@kmelmon nice, I thought one could only use Runs/Inlines in RichTextBlock. Good thing we don't own xaml text... oh wait... 😛